cobbler/templar.py

```diff
@@ -84,19 +89,17 @@ def check_for_invalid_imports(self, data):
                 if self.settings and rest not in self.settings.cheetah_import_whitelist:
                     raise CX("potentially insecure import in template: %s" % rest)
 
-    def render(self, data_input, search_table, out_path, subject=None, template_type=None):
+    def render(self, data_input: Union[TextIO, str], search_table: dict, out_path: Optional[str], subject=None,
```

Comment:
Since the subject is not used anymore and you removed the it in `render_cheetah()` and `render_jinja2()`, why not remove it here, too?

Reply:
Yes true. I will do it.